lib/screens/layout_settings_screen.dart

```diff
@@ -50,3 +54,30 @@ class LayoutSettingsScreen extends StatelessWidget {
     );
   }
 }
+
+class ShowArtistChipImageToggle extends StatelessWidget {
+  const ShowArtistChipImageToggle({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return ValueListenableBuilder<Box<FinampSettings>>(
+      valueListenable: FinampSettingsHelper.finampSettingsListener,
+      builder: (_, box, __) {
+        bool? showImage = box.get("FinampSettings")?.showArtistChipImage;
+
+        return SwitchListTile.adaptive(
+          title: Text(AppLocalizations.of(context)!.showArtistChipImage),
+          value: showImage ?? false,
+          onChanged: showImage == null
+              ? null
+              : (value) {
+                  FinampSettings finampSettingsTemp =
+                      box.get("FinampSettings")!;
+                  finampSettingsTemp.showArtistChipImage = value;
+                  box.put("FinampSettings", finampSettingsTemp);
+                },
+        );
+      },
+    );
+  }
+}
```

Comment:
Sure, I can move it.  I personally prefer to keep all these settings widgets in the screen file though.  It makes them easier to access, and it's not like they're going to get re-used anywhere.

Reply:
I don't have a strong preference either way, I just want to keep things consistent.  

If you feel like refactoring all settings to have  them in the same file instead, feel free to submit a PR ^^

Reply:
It's already a bit inconsistent because I used this style for all the settings I've already added/moved.  regardless, I've put this into a separate file for now.